topics/homework/week2/task_1.py

```diff
@@ -0,0 +1,66 @@
+#Bubble Sort Algorithm
```

Comment:
1 PEP 8: E265 block comment should start with '# ' (space is needed after #)
2 It is better to use __doc__ instead of comment
def check_scenario():
""" Function does following... """
<your code>
...

print(check_scenario.__doc__)

Reply:
Spaces are added